.ddev/commands/cypress/percy-run

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#ddev-generated
```

Comment:
@zeshanziya, is this actually ddev-generated? If it's copy/pasted, please remove this line.